### DISCUSSION TITLE
How to transfer holdings from one account to another

### DISCUSSION THREAD

**Discussion by nahsi** [ghostfolio/ghostfolio#3113] [Q&A]
Imagine I have "Nexo" account that has 1 ETH and I have "Ledger" account that has 0 ETH.
How can I transfer 1 ETH from "Nexo" to "Ledger"?

The ways I found:
- sell ETH on "Nexo" and buy on "Ledger"
- delete all the transactions related to "Nexo" and create them with account "Ledger"
- maybe edit db manually?

**Comment by dtslvr** [comment]
Here is a potential solution: https://github.com/ghostfolio/ghostfolio/discussions/1899#discussioncomment-6758311
